src/app/admin/core/unrelease.service.ts

```diff
@@ -33,6 +33,25 @@ export class UnreleaseService {
       .toPromise();
   }
 
+  /**
+   * Deletes a revision of a learning object. This is designed to allow an editor to create a new
```

Comment:
I don't know if this is just the way I am seeing it, but you could make combine the first two comments together because they kind of look broken apart. For example, you could try this:

* Deletes a revision of a learning object. This is designed to allow an editor to create a new revision when it is necessary for the editorial process to continue.

OR
* Deletes a revision of a learning object. This is designed to allow an editor
* to create a new revision when it is necessary for the editorial process to continue.